index.html

```diff
@@ -12,74 +12,235 @@
+            <button type="button" class="btn">Get my CV</button>
+
+            <section class="skils-container2">
+                <div class="cont">
+                    <div class="skill-list">
+                        <span>language</span>
+                        <span class="material-symbols-outlined">
+                            expand_more
+                        </span>
+                    </div>
+                    <ul>
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>Html</li>
+                        <li>Css</li>
+                    </ul>
+                </div>
+                <div class="cont">
+                    <div class="skill-list">
+                        <span>Frameworks</span>
+                        <span class="material-symbols-outlined">
+                            expand_more
+                        </span>
+                    </div>
+                    <ul class="list">
+                        <li>Bootstrap</li>
+                        <li>Ruby On Rials</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+                <div class="cont">
+                    <div class="skill-list">
+                        <span>Skills</span>
+                        <span class="material-symbols-outlined">
+                            expand_more
+                        </span>
+                    </div>
+                    <ul class="list">
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                </div>
+            </section>
+
+            <div class="social">
+                <a href="https://github.com/MostafaShahat2021" rel="noopener noreferrer nofollow" target="_blank">
+                    <img src="/img/git.png" alt="GitHub-Profile">
+                </a>
+                <a href="https://www.linkedin.com/in/mostafa-shahat-a75810208/" rel="noopener noreferrer nofollow"
+                    target="_blank">
+                    <img src="/img/linkedin.png" alt="LinkedIn-profile">
+                </a>
+                <a href="#" rel="noopener noreferrer nofollow" target="_blank">
+                    <img src="/img/Angelist.png" alt="">
+                </a>
+                <a href="https://twitter.com/Mostafa99738811" rel="noopener noreferrer nofollow" target="_blank">
+                    <img src="img/Vectotr.png" alt="twitter">
+                </a>
+                <a href="#" rel="noopener noreferrer nofollow" target="_blank">
+                    <img src="img/Vector1.png" alt="">
+                </a>
+                <img class="hr" src="img/Vector-4.png" alt="hrl">
+            </div>
+        </article>
```

Comment:
- [ ] This About section is one with the Home page in Figma template # 4. Kindly delete this section to look similar to the Figma template.